build-support/travis/travis.yml.mustache

```diff
@@ -0,0 +1,330 @@
+# GENERATED, DO NOT EDIT!
```

Comment:
It's slightly confusing to have this here. Consider templating in the warning and suggestion of which filename to edit?

Reply:
I see what you mean, will do.